content/en/docs/tasks/traffic-management/egress/egress-gateway/index.md

```diff
@@ -474,6 +474,13 @@ external service.
     EOF
     {{< /text >}}
 
+{{< warning >}}
```

Comment:
The warning needs to be indented 4 spaces, are it will mess up the numbering of the following steps.